src/apps/answers/db/schemas.py

```diff
@@ -74,6 +76,7 @@ class AnswerItemSchema(Base):
     migrated_data = Column(JSONB())
     assessment_activity_id = Column(Text(), nullable=True, index=True)
     tz_offset = Column(Integer, nullable=True, comment="Local timezone offset in minutes")
+    reviewed_submit_id = Column(UUID(as_uuid=True), nullable=True)
```

Comment:
Please rename to `reviewed_flow_submit_id ` for better understtanding, as activity answer also has submit_id, not only flow submission